ipn/ipnlocal/peerapi.go

```diff
@@ -401,6 +401,10 @@ func (h *peerAPIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		h.handlePeerPut(w, r)
 		return
 	}
+	if r.URL.Path == "/v0/goroutines" {
```

Comment:
Why not a full-blown pprof http handler? (I'm only half joking.)

Reply:
turns out that's useless, at least for iOS which is what we want this for most:

* CPU sampling doesn't work on iOS/macOS due to how we call into Go from C (confuses backtraces?)
* we don't have the memory on iOS to keep the memory profile hashtable